x-pack/legacy/plugins/maps/server/maps_telemetry/telemetry_task.js

```diff
@@ -10,7 +10,14 @@ const TELEMETRY_TASK_TYPE = 'maps_telemetry';
 
 export const TASK_ID = `Maps-${TELEMETRY_TASK_TYPE}`;
 
-export function scheduleTask(server, taskManager) {
+export function scheduleTask(server) {
+  const taskManager = server.plugins.task_manager;
+
+  if (!taskManager) {
+    server.log(['warning', 'telemetry'], `Task manager is not available`);
```

Comment:
I dont think we should be logging a warning if the task manager is not available, maybe a `debug`